### PR TITLE
vp66xx wifi

### PR DESCRIPTION
**Pull request by pkubaj**

### CODE REVIEW

File: docs/variants/protectli_vp66xx/post-install.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Post-installation setup
+
+This document contains extra steps to perform after installing Dasharo in order
+to enable full functionality.
+
+## Windows
+
+This section covers Windows post-install steps tested on Windows 11. It is
+likely that similar procedures would for others Linux distributions as well.
```
Comment (macpijan) [inline docs/variants/protectli_vp66xx/post-install.md:9]:
```suggestion
likely that similar procedures would <missing work here?> for others Linux distributions as well.
```

Comment (macpijan) [reply docs/variants/protectli_vp66xx/post-install.md:9]: Why it is likely for Linux distros, if it happens on Windows? There was any problem with Ubuntu?

File: docs/variants/protectli_vp66xx/post-install.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+### Qualcomm Atheros cards may be missing driver
+
+If you experience Windows to fail to recognize Qualcomm Atheros WiFi cards,
+you may need to install the necessary driver manually.
+
+Download the Windows driver from https://www.ath-drivers.eu/download-driver-nr-382-for-atheros-QCA6174A-and-Windows10.html
```
Comment (macpijan) [inline docs/variants/protectli_vp66xx/post-install.md:16]: Can you use md syntax to hide the link? E.g. [Windows driver](link).

File: docs/variants/protectli_vp66xx/post-install.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Using Device Manager, select Update Driver for Network Device located in
```
Comment (macpijan) [inline docs/variants/protectli_vp66xx/post-install.md:17]: Please use backticks, or other means, to emphasize the names of the menus and options from the rest of the text.

**Review by macpijan — CHANGES_REQUESTED** [review]

File: docs/variants/protectli_vp66xx/post-install.md (hunk 1 above)
Comment (pkubaj) [reply docs/variants/protectli_vp66xx/post-install.md:9]: No, that was a leftout sentence from the vp46xx's post-install.

File: docs/variants/protectli_vp66xx/post-install.md (hunk 2 above)
Comment (pkubaj) [reply docs/variants/protectli_vp66xx/post-install.md]: Done.

File: docs/variants/protectli_vp66xx/post-install.md (hunk 3 above)
Comment (pkubaj) [reply docs/variants/protectli_vp66xx/post-install.md]: Done.

**Review by macpijan — APPROVED** [review]

**Pull request merged**